app/configuration/Config.scala

```diff
@@ -127,16 +130,15 @@ object Config {
         prpIds.digitalPackQuaterly,
         prpIds.digitalPackYearly
       )),
-      campaignName = "DigiPack - 17% off for 3 months",
-      codes = PromoCodeSet(PromoCode("17OFF")),
-      description = "Get 17% off for 3 months when you subscribe.",
+      campaignName = s"DigiPack for just £9.99 a month (~17% discount)",
+      codes = PromoCodeSet(PromoCode("DPA30")),
+      description = "For a limited time you can enjoy the digital pack for just £9.99 a month (usually £11.99). Get every paper delivered to your tablet for less than 35p an edition, plus an ad-free experience on your live news app.",
       expires = new LocalDate(2016,4,1).toDateTime(LocalTime.Midnight, DateTimeZone.forID("Europe/London")),
-      imageUrl = "https://media.guim.co.uk/9ee88fc2f08bc23e69e2e11a4d4964f4120c6725/0_0_850_418/850.jpg",
-      redemptionInstructions = "We'll send redemption instructions to your registered email address",
-      roundelHtml = "Free <span class='roundel__strong'>17%</span> off",
+      imageUrl = None,
+      roundelHtml = "Only £9.99 a month</span><span class='roundel__byline'>usually £11.99",
```

Comment:
We will need to consider some better ways to customise HTML in the promos. Stuffing a lot of HTML into strings feels uncomfortable enough, but in this particular case it's very tightly coupled to the template it gets inserted into (i.e. tags whose opening/closing pair is outside this string). This will definitely cause maintenance headaches later